index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+  <head>
+    <title></title>
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+  </head>
+
+  <body>
+    <nav id="navbar" style="position:sticky;top:0;">
+      <a href="#welcome-section">Welcome Section</a>
+    </nav>
+    <style type="text/css">
+      @media only screen and (max-width: 600px) {
+        #welcome-section {
+          background-color: #000000;
+          color: #ffffff;
+        }
+      }
+    </style>
+
+
+    <section id="welcome-section">
+      <h1>Hi! I am a Software Developer</h1>
+    </section>
+    <section id="projects">
+      <a id="profile-link" class="project-tile" href="" target="_blank"><img class="project-image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" "title=" project"></img>
```

Comment:
could you please add an "alt" attribute? The alt attribute provides alternative information for an image if a user for some reason cannot view it.